aspnetcore/security/cookie-sharing.md

```diff
@@ -49,6 +49,8 @@ services.ConfigureApplicationCookie(options => {
 });
 ```
 
+**Note:** The preceding instructions don't work with `ITicketStore` (`CookieAuthenticationOptions.SessionStore`). This approach doesn't work because  the session id claim set by Owin has the type string `Microsoft.Owin.Security.Cookies-SessionId` while ASP.Net Core expects `Microsoft.AspNetCore.Authentication.Cookies-SessionId`. For more information, see [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/21163).
```

Comment:
```suggestion
**Note:** The preceding instructions don't work with `ITicketStore` (`CookieAuthenticationOptions.SessionStore`).  For more information, see [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/21163).
```

If we're not going to provide a solution here then we don't need the implementation details.